README.md

```diff
@@ -1 +1,36 @@
-# hw01
+# hw01 - My Bio
+Hi, this is Zewei (Whiskey) Liao, a third-year Sociology Major and an international student from Shenzhen, China, a city right next to Hong Kong. 
+
+## The Origin of "Whiskey"
+Zewei is my original name in Chinese but since a lot of people (i.e. non-Chinese speakers) are having a hard time pronouncing it (and messing it up really badly into a word that sounds both hilariously and awkwardly inappropriate in Chinese -- see the bottom of this section), I usually just go by **Whiskey**, a nickname I picked for myself in primary school when I was crazy about a Japanese anime series in which all the antagonists, who are basically all gang members, are given code names based on alcoholic beverages. Yes, there are a wide variety of them, which include:
+- Gin
+- Vodka
+- Vermouth
+- Tequila
+- and basically all sorts of Whiskey (except for "Whiskey" itself), including:
+  - Bourbon
+  - Scotch
+  - Irish
+  - Rye
+
+Since none of them is actually "Whiskey," and the first syllable of "Whiskey" sounds very close to the last character of my name ("Wei" from "Zewei") both in Chinese and in English, I assumed my membership of the group with this coolest name.
+
+Check this [wiki page](https://www.wikiwand.com/en/List_of_Case_Closed_characters#Black_Organization) for a more comprehensive list of my fellows! 
+
+<details>
+  <summary> <i>Click here to see what "Zewei" sounds like in Chinese when it's mispronounced: </i> </summary>
+  Ziwei, which is what "Zewei" sounds like when mispronounced, means masturbation in Chinese
+  
+</details>
+
+## Some other things about me
+- I'm a decade-long Lakers fan :basketball: :trophy:
+- I also love Barcelona but I really have no idea what they are doing these days... :soccer: :man_facepalming:
+- I also played ping pong, pool, and volleyball (ranked in decreasing order of skills) :ping_pong: :8ball: :volleyball:
+- I'm a new transfer student from Wesleyan University, a small liberal arts college in Connecticut where I was enrolled in a selective pre-law program until I realize law school might not be the thing for me and I really wanna explore more about (quantitative) social science research, hence my transfer :man_student: :man_technologist:
+- I farmed on-and-off during weekends for a year, and for 2 months during the summer on the student-run organic farm in my previous college :man_farmer:	
+- I'm currently exploring stand-up comedy as a genre of humor, but man, the cultural and language barrier is real! :studio_microphone:	:man_shrugging:
```

Comment:
I cannot understand humor in my second language either. Irony in English seems especially hard to pick up, though.